plugins/pencilblue/templates/angular/admin/elements/url_safe.html

```diff
@@ -3,17 +3,20 @@
 		require: 'ngModel',
 		link: function(scope, element, attrs, modelCtrl) {
 			modelCtrl.$parsers.push(function (inputValue) {
-				if(!inputValue.length) {
-					return inputValue;
-				}
+				
+				if(inputValue){
+					if(!inputValue.length) {
+						return inputValue;
+					}
 
-				var transformedInput = inputValue.replace(/[^a-zA-Z0-9_-]/g, '');
-				if (transformedInput!=inputValue) {
-					modelCtrl.$setViewValue(transformedInput);
-					modelCtrl.$render();
-				}
+					var transformedInput = inputValue.replace(/[^a-zA-Z0-9_-]/g, '');
+					if (transformedInput!=inputValue) {
+						modelCtrl.$setViewValue(transformedInput);
+						modelCtrl.$render();
+					}
 
-				return transformedInput;
+					return transformedInput;
+				}
```

Comment:
This modification will cause things to break if inputValue is null or undefined, because it'll never hit a return statement. We must always return a value.